dark-mode.css

```diff
@@ -573,6 +573,11 @@ body.dark-mode button.rcx-contextual-message__follow + div.rcx-box--full {
 body.dark-mode .rcx-option__content {
 	color: var(--color-gray-light);
 }
+
+body.dark-mode .rcx-option--focus, .rcx-option:hover {
+    background-color: var(--rc-color-primary-dark);
```

Comment:
@TBG-FR For me this rule still doesn't work, it's not overriding the default rule. We could override the default rule with an `!important` here, but I think I found another solution. Here's the regular rule:
```css
.rcx-option--focus, .rcx-option:hover {
    background: #f7f8fa;
    background: var(--rcx-color-neutral-100,#f7f8fa);
}
```
We could instead overwrite this var in our dark mode vars section:
```css
body.dark-mode {
    /* ... */
    --rcx-color-neutral-100: var(--rc-color-primary-dark);
    /* ... */
}
```
Let me know if this works for you and what you think!

Reply:
You're right, looks like I missed something while testing that fix... Yours works perfectly well, I'll fix this right now !